Adding topic "Localizing igEditors"

dafo opened the pull request.

dafo committed:
Adding a new topic - "Localizing igEditors"

dafo committed:
Adding Realted topics

dafo committed:
Changing the apostrophe

dafo commented:
This should not be merged as more information must be provided in the new topic.

dafo committed:
Adding information to the Localizing igEditors topic.
Adding a link to the new topic inside the Configure editors topic.

kdinev commented on topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md:
```diff
@@ -0,0 +1,108 @@
+
+Localization resources for the controls are in Bulgarian, Russian, Japanese, German, Spanish and French languages. These reside in js/modules/i18n (where js is the root folder for the JavaScript files in the Ignite UI program installation path). To get more into using the localization resources you can read the ["Customizing the Localization of Ignite UI Controls"](customizing-the-localization-of ...
+
+The regional settings, on the other hand, provide localized formats including dates, numbers, and currency symbols for the editors. These reside in the ../js/modules/i18n/regional (where js is the root folder for the JavaScript files in the Ignite UI program installation path).
```
provide region specific formats for dates, numbers, as well as currency symbols, floating point symbols, decimal separators, default decimal rounding, etc.

kdinev commented on topics/02_Controls/igEditors/Config/03_Localizing_igEditors.md:
```diff
@@ -0,0 +1,108 @@
+
+## Topic Overview
+
+In this sample we show how a culture can be set to the editors' `regional` option to change the format of dates, times, numbers and currencies. Three regions are configured here (United States, Japan and Tamil, India) and there are many more to choose from in the infragistics.ui.regional-i18n.js files. 
```
change the default formatting of dates, numbers, and currencies.

dafo committed:
Suggested changes applied.

dafo committed:
Final improvements.

damyanpetev committed:
Update 03_Localizing_igEditors.md

Added month/day names and default patterns to date description.
Formatting of the options section (moved date + picker first)

Lipata committed:
Update datepicker and datepickerOptions regional config

damyanpetev committed:
Update 03_Localizing_igEditors.md

kdinev commented:
@dafo Is this ready for merge?

dafo commented:
@kdinev I think it is ready to be merged.
